scripts/docker-helper-4.x/cht-docker-compose.sh

```diff
@@ -159,9 +159,23 @@ get_nginx_container_id() {
   docker ps --all --filter "publish=${NGINX_HTTPS_PORT}" --filter "name=${projectName}" --quiet  2>/dev/null
 }
 
-get_is_container_running() {
+nginx_is_running() {
```

Comment:
suggestion: `is_nginx_running` sounds a little better than `nginx_is_running`, which sounds declarative rather than a check. 😅

Reply:
ah - good call! fixed in 0031c95c5